src/Shared/constants.ts

```diff
@@ -24,6 +24,14 @@ export const SERVER_SUBEXT = ".server";
 export const CLIENT_SUBEXT = ".client";
 export const MODULE_SUBEXT = "";
 
+export const FILENAME_WARNINGS = new Map();
```

Comment:
If we do keep this map, it should be `new Map<string, string>()`

Reply:
Odd that didn't get picked up by noImplicitAny or something